### DISCUSSION TITLE
Quais as 3 próximas coisas que você quer aprender ?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1490] [Discussão]
Inspirado em uma das perguntas feitas aqui no fórum, queria saber o que vocês querem aprender em um futuro próximo.

**Comment by romulosardinha** [comment]
eu quero estudar mais: 

TDD
Node
Integração de pagamentos

**Comment by synga** [comment]
Cara é tanta coisa hahahaaha mas o que to querendo mais é:

NestJS (que estou estudando e gostando MUITO)
RxJS
Essas coisas de docker, parte de devops que entendo bem pouco

**Comment by adamfoerster** [comment]
TDD
API do Google
bash script

**Comment by gabrielelizio** [comment]
React JS, React Native e Node,

**Comment by dlfigueiredo** [comment]
React
Vue.js
NestJS

**Comment by ninetails** [comment]
Nanosserviços
Orquestradores de conteiners
Go(lang)

**Comment by onlurking** [comment]
- TDD ([Vue Test Utils](https://vue-test-utils.vuejs.org) com [Jest](https://jestjs.io/))
- [Vanilla JS](https://learnvanillajs.com/roadmap)
- [CSS Grid Layout](https://www.origamid.com/projetos/css-grid-layout-guia-completo/)

**Comment by Marcos1305** [comment]
Node
TDD
React

**Comment by jair-anderson-souza** [comment]
Node
CSS
SEO

**Comment by isacmoura** [comment]
Nossa, muita coisa veio na minha mente, mas neste momento:

Elastic
TDD
Flask

**Comment by yuritoledo** [comment]
@ninetails com nanoservices, você diz FaaS e coisas do gênero?

**Comment by ninetails** [comment]
@yuritoledo sim, _lambdas_ e gerenciamento usando alternativas a GCF/AWS Lambda/Azure Functions, tipo Kubeless e arquitetura envolvida...

mas é só desejo, não sei se irei ver algo tão cedo pq frontend tá bem bacana e ainda tem muita coisa pra aprender e sempre surge algo novo

**Comment by viniciusreiss** [comment]
Node, 
Testes,
Flutter

**Comment by renedocket** [comment]
CSS Grid Layout 
TDD
ReactJS

**Comment by eliseumds** [comment]
* Fazer o setup de uma empresa em algum canto com impostos baixos como Dubai ou Cyprus pra poder trabalhar remoto, além de não ter que se lascar com IOF, taxas abusivas, e um sistema bancário depredado
* Topic modelling (extrair palavras-chave de texto usando machine learning)
* Como fazer migração de banco de dados decentemente no Kubernetes

@isacmoura e outros, se estiverem afim de aprender ElasticSearch, eu posso ajudar. Eu fiz a parte de busca do site https://www.productreview.com.au/c/cars (saca só os filtros na lateral). Agregações, árvores de categorias, scripted fields, data normalisation, autocomplete, uns mappings furiosos e etc. Mais de 2,5 milhões de itens no banco espalhados em 750 categorias diferentes com mais de 3000 campos.

**Comment by eliseumds** [comment]
@ninetails algum progresso com Kubeless? Adoro Kubernetes, uso o [Serverless framework](https://serverless.com/) mas não sei se vale tanto a pena. Parece que a experiência com Kubeless é melhor, mas vai saber... tem tantas variáveis envolvidas.

**Comment by ninetails** [comment]
@eliseumds pra falar a verdade é meio um roadmap pra longo prazo, atualmente ando mais focado em coisas de Front, infelizmente ainda não tive muita oportunidade pra estudar k8/kubeless

**Comment by barraponto** [comment]
* Smart contracts (blockchain)
* Machine Learning
* Eletrônica/Robótica (a parte hardware, o Arduino eu já brinquei bastante)

**Comment by wilsonneto-swe** [comment]
- Azure Dev Ops
- ReactJS
- React Native

**Comment by vagner531** [comment]
- Node
 - React JS
 - React Native
